pennylane/measurements/measurements.py

```diff
@@ -142,10 +142,13 @@ def __init__(
         self.id = id
         self.log_base = log_base
 
-        if wires is not None and obs is not None:
-            raise ValueError("Cannot set the wires if an observable is provided.")
+        if wires is not None:
+            if len(wires) == 0:
+                raise ValueError("Cannot set an empty list of wires.")
+            if obs is not None:
+                raise ValueError("Cannot set the wires if an observable is provided.")
 
-        self._wires = wires or Wires([])
+        self._wires = wires
```

Comment:
If `wires = None` we keep `self._wires = None` to avoid raising an error when calling `__copy__`.

From outside the class we still set wires to `Wires([])` if wires is `None.

Reply:
Might be worth putting that in a comment here.  Or somewhere. Basically along the lines of `_wires is None indicates broadcasting across all available wires.  It translates to the public property wires = Wires([])`.

Reply:
Done!